packages/eslint-config-migme/preact.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [
+    './base',
+    './rules/strict',
+    './rules/preact',
```

Comment:
just curious, would it be okay to specified as following 
```
module.exports = {
  extends: [
    './base',
    './rules/strict',
    './rules/react',
    './rules/preact',
  ].map(require.resolve),
  rules: {},
}
```
and in `/preact.js` only set as following

```
 "settings": {
    "react": {
      "createClass": "h", // Regex for Component Factory to use, default to "createClass"
      "pragma": "preact",  // Pragma to use, default to "React"
      "version": "15.0" // React version, default to the latest React stable release
    }
  },
```